### ISSUE TITLE
Is `amdgpu-install`ing the latest AMD drivers worth doing?

### ISSUE THREAD

**Issue by edmundito** [s-damian/thinkpad-t14-gen-5-amd-linux#1]
Thank you so much for making this available! I was able to fix my wifi intermittent issues on my P14s Gen5. 

I was wondering if you've experimented with the latest [AMD Radeon drivers](https://www.amd.com/en/support/download/linux-drivers.html). I tried using `amdgpu-install` on Ubuntu 24 LTS to upgrade the drivers but it seemed to have made everything worse (lost resolution and refresh rate options) and I had to uninstall. Maybe not worth pursuing?

**Comment by s-damian** [comment]
Hello,
Thanks for your comment.
I don't quite understand your problem. What problem did you try to install the AMD Radeon drivers to try to solve? The brightness problem ("Fix automatic brightness reduction")? (it has nothing to do with the wifi problem).

"I was wondering if you've experimented with the latest AMD Radeon"
__No. I haven't tried (I'm using Mint 22, which is based on Ubuntu 24.04).

**Comment by s-damian** [comment]
Please,
Also, can you tell me, among these 4 tutorials:
https://github.com/s-damian/thinkpad-t14-gen-5-amd-linux
Which ones worked for the Laptop "P14s Gen5"?
Like this I will add at the top of my tutorials that it also works for this laptop model.
Thanks.

**Comment by s-damian** [comment]
Thanks

**Comment by erny** [comment]
@edmundito I have installed [AMD Radeon drivers](https://www.amd.com/en/support/download/linux-drivers.html) amdgpu-install in my P16s Gen 2 (AMD Radeon 780M) and it resolved display wake up after suspend / wakeup. It also makes the GPU available to use for AI LLM models (use "sudo rocminfo"). The think is that I had to disable "UEFI Secure boot" in BIOS to make "hibernate / wakeup" available. Together with the network fix, I have now nearly everything working. (touch screen, wifi, dual display, AMD temperature sensors, ...)
